packages/framework/composition/README.md

```diff
@@ -1 +1,47 @@
-# @fluentui-react-native/use-styling
+# @fluentui-react-native/composition
```

Comment:
Did the actual package name change or is this just a documentation update?

Reply:
I think this was a copy paste error, the actual name of the package is composition. use-styling is a different package.